processing/data_collection/gazette/pipelines.py

```diff
@@ -93,40 +94,50 @@ def doc_source_text(self, item):
         with open(text_path, "r") as f:
             return f.read()
 
+    def txt_source_text(self, item):
+        """
+        Gets the text from txt files
+        """
+        with open(
+            os.path.join(FILES_STORE, item["files"][0]["path"]), encoding="ISO-8859-1"
+        ) as f:
+            return f.read()
+
     def is_pdf(self, filepath):
         """
-        If the file path ends with pdf returns True. Otherwise,
+        If the file type is pdf returns True. Otherwise,
         returns False
         """
-        return self.get_extension(filepath) == "pdf"
+        return self._is_file_type(filepath, file_types=["application/pdf"])
 
     def is_doc(self, filepath):
         """
-        If the file path ends with doc,  docx or odt returns True. Otherwise,
+        If the file type is doc or similar returns True. Otherwise,
         returns False
         """
-        extension = self.get_extension(filepath)
-        return extension == "doc" or extension == "docx" or extension == "odt"
+        file_types = [
+            "application/msword",
+            "application/octet-stream",
```

Comment:
I think you can remove the `application/octet-stream`. It's to generic. You can also add `application/vnd.openxmlformats-officedocument.wordprocessingml.document`, it's the docx mime type. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types

Reply:
Gotcha! 

I don't remember for which file type I've added `application/octet-stream`. Maybe there was something wrong with the file sample that I downloaded to test it.

Thanks for the review!